inabox/README.md

```diff
@@ -155,6 +175,13 @@ TRACE[10-12|22:02:13.376] [batcher] AggregateSignatures took       duration=10.6
 TRACE[10-12|22:02:13.376] [batcher] Confirming batch...            caller=batcher.go:198
 ```
 
+To dispense that same blob (replace `$REQUEST_ID` with the request ID from the prior step):
```

Comment:
To **check the status of** that same blob

Reply:
Change made.
```suggestion
To check the status of that same blob (replace `$REQUEST_ID` with the request ID from the prior step):
```